src/FixedDataTableCellGroup.js

```diff
@@ -89,7 +89,7 @@ var FixedDataTableCellGroupImpl = createReactClass({
     for (var i = 0, j = columns.length; i < j; i++) {
       var columnProps = columns[i].props;
       var recycable = columnProps.allowCellsRecycling && !isColumnReordering;
-      if (!recycable || (
+      if (!recycable || columnProps.fixedRight || (
```

Comment:
Did you have to remove this because you stopped positioning by props.left?

Reply:
Yes, however, now I recall the fix below where I default to 0 when there is no `left` prop defined and realized that I can avoid adding both of these changes by setting the default prop value for `left` to 0 in `FixedDataTableCellGroup`